packages/workflow/package.json

```diff
@@ -5,7 +5,7 @@
   "license": "MPL-2.0",
   "private": true,
   "scripts": {
-    "start": "cross-env NODE_ENV=development nodemon --exec ts-node -r tsconfig-paths/register src/index.ts",
+    "start": "cross-env NODE_ENV=development LANGUAGES='en' nodemon --exec ts-node -r tsconfig-paths/register src/index.ts",
```

Comment:
Same question as above. Also, `cross-env` dependency is missing in workflow service.